skiko/src/jvmTest/kotlin/org/jetbrains/skiko/paragraph/FontCollectionTest.kt

```diff
@@ -74,5 +74,6 @@ class FontCollectionTest {
             }
             assertEquals(refCnt, t1.refCount)
         }
+
```

Comment:
Unneeded.

Reply:
it's remnant of skija test, will remove.